theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -287,12 +295,12 @@ const CurveEditorPopover: React.FC<IProps> = (props) => {
     trackData.keyframes[index + 1].handles[1],
   ]
 
-  const isFilterSetBeingUsed = useRef(false)
+  const isFilterUpdatePrevented = useRef(false)
```

Comment:
Change: Add docs describing the UX you're going for, here. What kinds of filter update cases are you trying to prevent? What are some weird cases you're avoiding w.r.t. filtering / searching?

Reply:
Outdated after refactor